docker-compose.yml

```diff
@@ -1,4 +1,5 @@
 version: '3'
+name: koku
```

Comment:
This is causing issues for me :suspect: 

```
(koku) [lcouzens@fedora-desktop koku]$ make docker-up-min-no-build && make docker-logs
/usr/bin/docker-compose up -d db
ERROR: The Compose file './docker-compose.yml' is invalid because:
'name' does not match any of the regexes: '^x-'

You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
make: *** [Makefile:374: docker-up-db] Error 1

Reply:
That seems like you're using the legacy version of Docker Compose. What does `docker compose` version show? I'm guessing that failed and falls back to `docker-compose`.

Reply:
Thats a fair point, I probably do need to update stuff :shark: